src/heitech.zer0mqXt.core/Transport/Zer0mqMessageAttribute.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace heitech.zer0mqXt.core.Transport
+{
+    ///<summary>
+    /// Depicts a Message to be used with the Zer0mqXt library.
+    /// <para>The MessageId is used to check for matching Messages (Make also sure that you can serialize/deserialize the matching messages)</para>
+    ///</summary>
+    [System.AttributeUsage(System.AttributeTargets.All, Inherited = false, AllowMultiple = false)]
```

Comment:
I believe this attribute is currently only observed on types.
Should we restrict the target to `Class`, `Struct` and `Enum` only?

And now I am uncertain if it would actually work for `struct`s and `enum`s.

Reply:
Only to class then. Was taken from a snippet. Only classes are allowed for messaging.